### PR TITLE
External UCS parameters: Script 021

### PR DESCRIPTION
**Pull request by dboltovskyi**
Script is part of test set: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1169

### CODE REVIEW

File: test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+-- SDL must:
+-- a. consider this PTU as invalid
+-- b. do not merge this invalid PTU to LocalPT.
+--
+-- Steps:
+-- 1. Register app
+-- 2. Activate app
+-- 3. Perform PTU (make sure 'disallowed_by_external_consent_entities_off' section is defined
+-- with invalid parameters)
+-- 4. Verify status of update
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, If PTU finished with not UP_TO_DATE status merge won't happen

File: test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+local startLine = 1
+
+local function sequence(desc, updateFunc)
+
+  local tcName = "TC[" .. string.format("%02d", n) .. "] - [" .. desc .. "] "
+  tcName = tcName .. string.rep("-", 50 - string.len(tcName))
+
+  Test[tcName] = function() end
+
+  function Test.StopSDL()
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Agreed, but I tried to perform clean run for every case.
Test consists of 18 cases. Is it possible to perform so many attempts within 1 PTU?
Won't we have a issue with retry strategy etc.?

File: test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 1 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, I also agree with you, but we have a lot of scripts on Policies functionality. And such verification (unsuccessful update) is part of these scripts.
All these checks have to be included in scope of regression testing.

Comment (istoimenova) [inline test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @dboltovskyi, agree.

**Pull request merged**